src/Sulu/Bundle/TestBundle/Resources/dist/orm_mysql.yml

```diff
@@ -7,3 +7,4 @@ doctrine:
         user:     "%database_mysql_user%"
         password: "%database_mysql_password%"
         charset:  UTF8
+        server_version: 5.6
```

Comment:
Travis is actually running on 5.5, so it would be great if you could adapt that here. Afterwards this should be ready to merge IMO.